src/custom/pages/Swap/SwapMod.tsx

```diff
@@ -490,7 +446,7 @@ export default function Swap({ history, location, className }: RouteComponentPro
                 showMaxButton={false}
                 hideBalance={false}
                 fiatValue={fiatValueOutput ?? undefined}
-                priceImpact={showWrap ? undefined : priceImpact}
+                priceImpact={onWrap ? undefined : priceImpact}
```

Comment:
Already mentioned in the other PR, we should keep `showWrap` here, unless it has been removed.

Reply:
Fixed in the next PRs